services/frontend/src/modules/news/pages/news.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react'
+import {
+  Box,
+  Tabs,
+  TabList,
+  TabPanels,
+  Tab,
+  TabPanel,
+  Heading,
+  Text,
+  Container,
+  Wrap,
+} from '@chakra-ui/react'
+import { DefaultLayout } from '../../../components/DefaultLayout'
+
+// タブ用のprops
+const tabprops = {
+  _focus: { outline: 'none' },
+  _selected: { bg: 'blackAlpha.200', color: 'red' },
+}
+
+const Page = () => {
```

Comment:
ここではより具体的な命名にするのが良いです！

```suggestion
const NewsPage = () => {
```

合わせてファイル名も `NewsPage.tsx` に変更してもらえるとありがたいです！